cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,36 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  assertProductInCart(prodName) {
+    cy.get('tbody tr td:nth-child(2)')
+      .should('have.text', prodName);
```

Comment:
Consider using a more flexible assertion like `should('include.text', prodName)` instead of `should('have.text', prodName)`. This change can help avoid issues if there are additional spaces or text around the product name.